eth/gaspricemonitor.go

```diff
@@ -148,6 +151,18 @@ func (gpm *GasPriceMonitor) Stop() error {
 	return nil
 }
 
+func (gpm *GasPriceMonitor) SetMaxGasPrice(gp *big.Int) {
```

Comment:
This is currently missing the metrics recording call via the monitor package that was included in the [method definition](https://github.com/livepeer/go-livepeer/blob/0a72c0ac52b97b37bd2fb51c865ac3f2a3592bc3/eth/backend.go#L134) in backend.

Reply:
fixed in 91ceac9